index.html

```diff
@@ -7,10 +7,21 @@
 
     <link rel="stylesheet" href="style.css" />
     <script src="app.js" defer></script>
+    <script src="navbar.js" defer></script>
   </head>
   <body>
     <main>
-      <div class="book">
+      <!-- Header Section -->
+      <header>
+        <div class="logo"><p>Awesome Books</p></div>
+        <div class="navbar">
+          <div class="links"><a href="#list" id="list-link">List</a></div>
+          <div class="links"><a href="#add" id="add-link">Add new</a></div>
+          <div class="links"><a href="#contact" id="contact-link">Contact</a></div>
+        </div>
+      </header>
+      <!-- List Section -->
+      <div id="list" class="book">
         <h1>Awesome Books</h1>
         <table class="table">
```

Comment:
- [ ] The styling of your _books list rows_ background color does not conform to the database table alternate rows color, as can be seen in the original wireframe below. Kindly ensure to improve upon this aspect.

Your design:
![image](https://user-images.githubusercontent.com/12745474/153090114-cd287ef7-8b50-41f5-9497-7a1626c6ddbe.png)

**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/134323111-24a1a54d-374a-4197-bdc5-7ddbbcc89636.png)